src/conduit/components/header.cljs

```diff
@@ -27,9 +27,15 @@
     (when icon " ")
     label]])
 
-(rum/defc Header [r route]
-  [:nav.navbar.navbar-light
-   [:div.container
-    [:a.navbar-brand {:href "#/"} "conduit"]
-    [:ul.nav.navbar-nav.pull-xs-right
-     (map #(rum/with-key (NavItem route %) (:label %)) nav-items)]]])
+(rum/defc Header [r route current-user]
+  (let [user-nav-items (if current-user
+                         (->> {:label (str "Hi, " (:username current-user))
```

Comment:
i'm sure you could extract this out of the component definition. also, it could be good to move the definition of 'should item be rendered' to the nav-items data structure itself. this approach is more flexible. for example, you could have :display-for field with possible values :logged, :non-logged, :always. and have a function for building actual nav items list